packages/loaders/src/elements/Inline.tsx

```diff
@@ -20,8 +20,9 @@ const InlineTypingIndicator: React.FC<IStyledTypingSvgProps> = props => {
 };
 
 interface IInlineProps extends React.HTMLAttributes<SVGSVGElement> {
-  /* Width of the loader in px */
+  /** Sets the size proportionally based on width designation. Number is in pixels */
```

Comment:
I don't love this one. Maybe:

1. Sets the size. Number specifies width in pixels
2. Sets the width in pixels and scales the loader proportionally